src/main/java/core/basesyntax/Square.java

```diff
@@ -0,0 +1,23 @@
+package core.basesyntax;
+
+public class Square extends Figure {
+    private final int side;
+
+    public Square(String color,int side) {
```

Comment:
Creating a figure, don't pass expressions in the constructor. It's better to create a variable for 'side' and pass it to the constructor for better readability.